db/migrate/20230118195231_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: {to_table: 'users'}
```

Comment:
- [ ] Nice job trying to implement the suggested change but you missed the part where the reference should be implemented with the `author_id` as shown in the [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)